docs/WINDOWS.md

```diff
@@ -14,20 +14,28 @@ This guide explains how to download, configure and run Retro AIM Server on Windo
       <img alt="screenshot of a chrome prompt showing a blocked download" src="https://github.com/mk6i/retro-aim-server/assets/2894330/6bf2fd79-0a42-48b2-a695-d777259a3603">
     </p>
 
+   In some cases, Chrome may outright block the download. 
+   
+   <p align="center">
+     <img alt="screenshot of a chrome blocking multiple download attempts" src="https://github.com/ukozi/retro-aim-server/assets/3391773/b3a9f5fc-bc5e-4b00-bc73-f71083af816a">
+   </p>
+
    > While the binaries are 100% safe, you can avoid the security concern by [building the application yourself](./BUILD.md).
    We do not provide signed binaries because of the undue cost and complexity.
 
+   You can alternatively download the file using a Powershell command: `Invoke-WebRequest -Uri "<insert link to version you wish to download here>" -OutFile "retro_aim_server.zip"` and exclude the file or directory from Defender.
```

Comment:
Would you please put the command in a code block? Like:

```powershell
`Invoke-WebRequest -Uri "<insert link to version you wish to download here>" -OutFile "retro_aim_server.zip"
```

Makes it a bit easier for people to see and copy/paste.

Reply:
Code block can start with ```powershell to do ps syntax highlighting

Reply:
Added a powershell code block.